unittests/tools/test_hcl_appscan_parser.py

```diff
@@ -37,3 +37,10 @@ def test_issue_9279(self):
         self.assertEqual(findings[5].mitigation, "Remediation: fix_61771\nAdvisory: attReferrerPolicyHeaderExist")
         self.assertEqual(findings[1].description, "Issue-Type:attHttpsToHttp\nThreat-Class: catInformationLeakage\nEntity: 7089695691196187648\nSecurity-Risks: sensitiveNotOverSSL\nCause-Id: sensitiveDataNotSSL\n")
         self.assertEqual(findings[10].cwe, 1275)
+
+    def test_issue_10074(self):
+        my_file_handle = open("unittests/scans/hcl_appscan/issue_10074.xml")
+        parser = HCLAppScanParser()
+        findings = parser.get_findings(my_file_handle, None)
+        my_file_handle.close()
+        self.assertEqual(4, len(findings))
```

Comment:
Can you add a check here to ensure the severity fix is working as expected as well?

Reply:
Done, please review again @cneill